### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by Stephen-Adom**
Vet clinic database: create animals table
In this pull request, the following tasks were implementated:
- Use the template to generate the repository.
- Created a database named vet_clinic.
- Create a table named animals with the following columns:
    `id`: integer
    `name`: string
    `date_of_birth`: date
    `escape_attempts`: integer
    `neutered`: boolean
    `weight_kg`: decimal
- Declared the sql command for creating database in `schema.sql` file
- Declared the sql commands for inserting data in `data.sql` file
- Declared the sql commands for quering data in `queries.sql` file

Find all animals whose name ends in "mon"

List the name of all animals born between 2016 and 2019

List the name of all animals that are neutered and have less than 3 escape attempts.

List the date of birth of all animals named either "Agumon" or "Pikachu".

List name and escape attempts of animals that weigh more than 10.5kg

Find all animals not named Gabumon.

Find all animals that are neutered.

Find all animals with a weight between 10.4kg and 17.3kg

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,3 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animal (id BIGSERIAL, name VARCHAR(50), date_of_birth DATE, escape_attempts INT, neutered BOOL, weight_kg DECIMAL);
```
Comment (pinkmoon25) [inline schema.sql]: - [ ] [OPTIONAL] Good job making the id column autoincremented, just one suggestion you can add the primary constraint and use the ID coulmn as Primary key :+1:

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animal WHERE name LIKE '%mon';
+
+SELECT name from animal WHERE date_of_birth  >= '2016-01-01' AND date_of_birth <= '2019-01-01';
```
Comment (pinkmoon25) [inline queries.sql]: - [ ] I think If you want to select all animals born between 2016 and 2019, the end date of the range should be December 31, 2019. The correct SQL query would be: `SELECT name FROM animal WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
`

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Stephen-Adom , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
gitflow :heavy_check_mark: 
Data and schema :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Stephen-Adom) [reply schema.sql]: Hello, @PinkMoon25 thank you for this out for me

File: queries.sql (hunk 2 above)
Comment (Stephen-Adom) [reply queries.sql]: @PinkMoon25 Thank you. Will make the required changes

**Review by gemmen29 — APPROVED** [review]
Hi @Stephen-Adom, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**